data/brands/amenity/school.json

```
@@ -151,6 +151,18 @@
         "operator:type": "private_non_profit"
       }
     },
+    {
+      "displayName": "Niles Township High School District 219",
+      "id": "",
+      "locationSet": {"include": ["us", "il"]},
```

```suggestion
      "locationSet": {"include": ["us-il.geojson"]},
```
For states, we use this format: `us-[state].geojson`
See https://github.com/osmlab/name-suggestion-index/wiki/Contributing#locationset-required (under 'States within the United States')

In the future, you can use https://location-conflation.com/?referrer=nsi to test out locationSets